eng/targets/Helix.targets

```diff
@@ -41,8 +41,11 @@ Usage: dotnet msbuild /t:Helix src/MyTestProject.csproj
     </PropertyGroup>
 
     <ItemGroup>
+      <_HelixAvailableTargetQueue Condition="'$(UseHelixOpenQueues)' == 'true'" Include="@(HelixAvailableTargetQueue)" />
+      <_HelixAvailableTargetQueue Condition="'$(UseHelixOpenQueues)' != 'true'" Include="@(HelixAvailableTargetQueue->'%(Identity)'->Replace('.Open',''))" />      
```

Comment:
nit: Odd naming (though this is an existing strangeness). What's especially "available" about these queues?

Reply:
This is part of the complexity that some things like IIS tests introduced where certain tests only target certain queues/platforms